SU-SWS/stanford_paragraph_types
# EARTH-295 Section Header Paragraph Type

# READY FOR REVIEW

# Summary
- Section Header Paragraph Type

# Needed By (Date)
- EOS

# Urgency
- Low

# Steps to Test

1. Checkout branch
2. `drush en stanford_paragraph_section_header -y`
3. create complex page with "Section Header" type
4. Verify it displays correctly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
